nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors/src/main/java/org/apache/nifi/processors/splunk/QuerySplunkIndexingStatus.java

```diff
@@ -194,7 +197,11 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                 final EventIndexStatusResponse splunkResponse = unmarshallResult(responseMessage.getContent(), EventIndexStatusResponse.class);
 
                 splunkResponse.getAcks().entrySet().forEach(result -> {
-                    final FlowFile toTransfer = undetermined.get(result.getKey());
+                    FlowFile toTransfer = undetermined.get(result.getKey());
+                    if (!extractBoolean(toTransfer.getAttribute(ACKCHECKED_ATTRIBUTE)).orElse(Boolean.FALSE)) {
+                        toTransfer = setAckCheckedToTrue(session, toTransfer);
+                        session.getProvenanceReporter().modifyAttributes(toTransfer, "ackChecked attribute has been modified to true.");
```

Comment:
I am not sure we need to store this as a provenance event (however it might not an issue)

Reply:
removed  event logging